baybe/acquisition/acqfs.py

```diff
@@ -228,6 +203,12 @@ class qLogNoisyExpectedImprovement(AcquisitionFunction):
     """Auto-prune candidates that are unlikely to be the best."""
 
 
+# Remapped because the log versions offer improved numerical stability
+ExpectedImprovement = LogExpectedImprovement
```

Comment:
I am actually not sure whether or not we should include this in the way it is implemented here. 

We basically *force* users to use the `log`-variant. Even if this claims to be better from a numerical point of view, this is not guaranteed, and also, maybe some users would prefer to use the non-`log` variant (for whatever reason). Also, if this change was a pure "no-brainer", then why didn't BoTorch just kick the old variants? Given that BoTorch likes to kick stuff out of their code, I am a bit worried if they handle this via a warning.

I would thus rather suppose to replace our defaults (if we do not already use the `log` variant there).

Reply:
I think this would equally do a reasonable job:
-  change or default from `qExpectedImprovement` to `qLogExpectedImprovement`
- mention the discouraged non-log functions as admonition somewhere linking to botorch
- this would un-require the sort of hidden changes

Reply:
Can do, but problem is the second point: there is simply no entry about acqfs in the user guide yet ... So how about we ignore that point for now (botorch does the discouraging job for us anyway) and do so once we update our user guide?

Reply:
Defaults are changed now. If you are OK with adding the note once we have an actual ACQF user guide, then thumbs up and I'll resolve + merge. If not, please suggest a better idea / tell me where you could imagine to have the note

Reply:
as already suggested the palce would be the recommender userguide, but if you dont want to add an admonition also fine for me because now theres no silent overwrite

Reply:
OK, since not strictly required, I'd rather skip for now. The recommender user guide doesn't quite seem the right place to me, I'd rather have a clean ACQF user guide where we perhaps also list the functions in a table with some context information (e.g. if they are to be used for active learning or function optimization, etc). There, we could add a little `*` next to the non-log functions and link it to an admonition displayed below the table or something.